ast_tools/passes/base.py

```diff
@@ -6,7 +6,7 @@
 
 __ALL__ = ['Pass', 'PASS_ARGS_T']
 
-PASS_ARGS_T = tp.Tuple[ast.AST, SymbolTable]
+PASS_ARGS_T = tp.Tuple[ast.AST, SymbolTable, dict]
```

Comment:
Unless the type actually has to be a `dict` (e.g. something being passed as globals to exec) annotation should be `MutableMapping`